lib/ts/recipe/thirdParty/index.ts

```diff
@@ -87,3 +88,5 @@ export let getUsersOldestFirst = Wrapper.getUsersOldestFirst;
 export let getUsersNewestFirst = Wrapper.getUsersNewestFirst;
 
 export let getUserCount = Wrapper.getUserCount;
+
+export let providers = thirdPartyProviders;
```

Comment:
Import statement should be (both of them should work)
- `import {Google, Facebook, Apple} from "supertokens-node/recipe/thirdparty";`
- `import ThirdParty from "supertokens-node/recipe/thirdparty";` and then `ThirdParty.Google`, `ThirdParty.Facebook`, etc

Reply:
> ```import ThirdParty from "supertokens-node/recipe/thirdparty";``` and then ```ThirdParty.Google```, ```ThirdParty.Facebook```, etc

Shouldn't this be
```import * as ThirdParty from "supertokens-node/recipe/thirdparty";``` and then ```ThirdParty.Google```, ```ThirdParty.Facebook```, etc

Reply:
No. What I had written is correct